drivers/hmis/app/models/hmis/ce/referral_enroller.rb

```diff
@@ -46,7 +46,9 @@ def create_enrollment(message)
       entry_date_errors.reject!(&:warning?)
       error_out(entry_date_errors.map(&:full_message).join(', ')) unless entry_date_errors.empty?
 
-      # TODO(#6709) - assign enrollment to unit
+      # TODO(#7537) - prevent conflicting unit occupancy
```

Comment:
Created https://github.com/open-path/Green-River/issues/7537 for this, take a look and feel free to add more detail/spec!